src/themes/eui-amsterdam/overrides/_call_out.scss

```
@@ -0,0 +1,8 @@
+.euiCallOut {
+  border-radius: $euiBorderRadius;
+  border-left: none;
+
+  .euiCallOutHeader__title {
+    font-weight: $euiFontWeightMedium;
+  }
+}
```

nit: Space at the end of doc
```suggestion
}

```